benchexec/tools/graves-par.py

```diff
@@ -0,0 +1,64 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.tools.coveriteam_verifier_validator as coveriteam
```

Comment:
Did you test this and does it work for you?
For me it produces `ImportError: No module named 'benchexec.tools.coveriteam_verifier_validator'`.

I guess something like https://github.com/sosy-lab/benchexec/blob/9cfa48ba87862d1bfa6da1c58b56583b8fdf7ff5/benchexec/tools/yogar-cbmc-parallel.py#L10 is needed.

Reply:
Thanks. Fixed